tests/unit/acquisition/test_rule.py

```diff
@@ -186,3 +190,33 @@ def test_trust_region_for_unsuccessful_local_to_global_trust_region_reduced() ->
     npt.assert_array_less(current_state.eps, previous_state.eps)  # current TR smaller than previous
     assert current_state.is_global
     npt.assert_array_almost_equal(current_state.acquisition_space.lower, lower_bound)
+
+
+class _BatchModelMinusMeanMaximumSingleBuilder(SingleModelAcquisitionBuilder):
+    def prepare_acquisition_function(
+        self, dataset: Dataset, model: ProbabilisticModel
+    ) -> AcquisitionFunction:
+        return lambda at: -tf.reduce_max(model.predict(at)[0], axis=-2)
```

Comment:
this will need to be a `BatchAcquisitionFunctionBuilder` rather than a `SingleModelAcquisitionBuilder`